README.md

```diff
@@ -160,16 +165,23 @@ To undo this change, use the `test_reverse` flag:
 
 ## Execution
 
-To execute the script and update Grouper and EDS, include the `portal`, `quota`,
-and `sync` command-line flags:
+To execute the script and update Grouper and EDS, include the `portal`, `quota`command-line flags:
 
 ```
 (figshare_patrons) $ ./scripts/script_run --config config/figshare.ini \
                        --persistent_path $persist_path \
                        --ldap_password $password --grouper_password $password \
-                       --quota --portal --sync
+                       --quota --portal 
+                       
+                     
 ```
 
+"--quota" will run quota for groups
+"--portal" will run portal for all grouper users. 
+
+The above is "dry run". 
+using "--sync" will execute the script!!! 
+
 Note: Without the `sync` flag, the above command line will perform a
 "dry run" where both `quota` and `portal` queries are conducted. It will
 indicate what Grouper updates will occur.
```

Comment:
Lines 185-187. Recommend keeping docs clean and avoiding redundant info. This same information is repeated above